sites/partners/pages/index.tsx

```diff
@@ -55,13 +55,20 @@ export default function ListingsList() {
 
   const columnDefs = useMemo(
     () => [
+      {
+        headerName: t("name"),
```

Comment:
Is the end result the same if we substitute `listings.name` here? If so, for consistency with the rest of the cols let's do that.

Reply:
So this was part of what was giving me trouble Friday.  Since it is a ListingDto, the name attribute is at the top level.  The ListingDto.listing didn't have a name attribute that I was able to trigger.  As a result if you switch it to `listings.name`, you get a column header that says "Listings.name".  I kept digging down the listings.name or listings.propertyAddress path, and its when I finally realized I need to be "up a level" on the DTO.